pep/streaming_connection.go

```diff
@@ -210,6 +210,14 @@ func (c *streamConn) tryConnect(addr string, tracer opentracing.Tracer, timeout
 	var conn *grpc.ClientConn
 	conn, err = grpc.DialContext(ctx, addr, opts...)
 	if err != nil {
+		c.lock.Lock()
+		defer c.lock.Unlock()
+
+		if c.state == scisClosing {
+			c.state = scisDisconnected
+			err = errStreamConnWrongState
+		}
+
```

Comment:
`WithBlock()` would throw an error if the connection to pdp could not be established https://pkg.go.dev/google.golang.org/grpc?tab=doc#DialContext

Reply:
The intention is to break the loop at https://github.com/infobloxopen/themis/blob/2ae78bb8e45be2c93f2bf9e109e0445395816b9b/pep/streaming_connection.go#L130-L134
This can happen only if `tryConnect()` returns `nil` or `errStreamConnWrongState`